lib/elixir_boilerplate_web/router.ex

```diff
@@ -1,6 +1,8 @@
 defmodule ElixirBoilerplateWeb.Router do
   use Phoenix.Router
 
+  @csp %{"content-security-policy" => "default-src 'self'"}
```

Comment:
Technically this is not a "CSP", this is a set of headers that are merged with Phoenix’s default secure headers. We might add other headers to it in the future. Therefore I think we should name it accordingly:

```suggestion
  @secure_headers %{"content-security-policy" => "default-src 'self'"}
```

What do you think?

Reply:
Definitively!

https://github.com/phoenixframework/phoenix/blob/v1.4.9/lib/phoenix/controller.ex#L1036-L1053